### PR TITLE
Fix #6374 - Render.com web builds fail

### LINKED ISSUE #6374: [Bug?]: Render.com web build fails due to only supporting yarn 1
### What's not working?

Web builds fail on Render.com

This is a known issue due to a lack of support for yarn 2/3 as noted here. https://feedback.render.com/features/p/add-support-for-yarn-2

In addition, after adding SKIP_INSTALL_DEPS env var to true the build still fails with another error.

After speaking with support changing the build cmd to 

yarn; yarn rw deploy render web

It fixes the issue and the build works. I will put a pull request in to add these details to the render.yaml file 

### How do we reproduce the bug?

Follow instructions here https://redwoodjs.com/docs/deploy/render and attempt to deploy to render 

### What's your environment? (If it applies)

_No response_

### Are you interested in working on this?

- [X] I'm interested in working on this

### PR DESCRIPTION
**Pull request by shansmith01**
See #6374 re Render.com web build fails due to only supporting yarn 1.

Updated the template file so others don't have the same issue

### CODE REVIEW

**Comment by noire-munich** [comment]
@simoncrypta I've assigned you as you are assigned to the issue - thought we should be consistent ( and maybe double check the bot?)

**Comment by simoncrypta** [comment]
@shansmith01 Thanks for that ! This cause so much trouble for everyone and this trick is not really well documented. So since we use yarn 3 by default, we should also skip dep sync for Render by default. Good job ! 

@noire-munich All right, I will check if we can update the bot for that. 🙂

**Review by simoncrypta — CHANGES_REQUESTED** [review]

File: packages/cli/src/commands/setup/deploy/templates/render.js [hunk 1]
```diff
@@ -15,11 +15,13 @@ services:
 - name: ${PROJECT_NAME}-web
   type: web
   env: static
-  buildCommand: yarn rw deploy render web
+  buildCommand: yarn; yarn rw deploy render web
```
[the remark this answers is not in the document]
Comment (shansmith01) [reply packages/cli/src/commands/setup/deploy/templates/render.js:18]: Makes sense to me

**Review by simoncrypta — APPROVED** [review]
Works like a charm, thanks a lot !

**Pull request merged**